templates/ces_client/index.html

```diff
@@ -0,0 +1,89 @@
+{% extends "base.html" %}
+{% load static %}
+{% block title %}Index{% endblock %}
+
+{% block extra_css %}
+    <link rel="stylesheet" href="{% static 'css/botui.min.css' %}" />
+    <link rel="stylesheet" href="{% static 'css/botui-theme-default.css' %}" />
+{% endblock %}
+
+{% block content %}  
+
+<div id="my-botui-app">
+  <bot-ui></bot-ui>
+</div>
+
+<div>
+    <textarea class="form-control"></textarea>
+</div>
+
+{% endblock %}
+
+{% block extra_javascript %}
+<script src="{% static 'js/lib/vue.min.js' %}" type="text/javascript"></script>
+<script src="{% static 'js/lib/botui.min.js' %}" type="text/javascript"></script>
+<script src="{% static 'js/lib/js.cookie.js' %}" type="text/javascript"></script>
+
+<script type="text/javascript">
+    $('document').ready(function(){
+        // Boilerplate code from Django documentation
+        // https://docs.djangoproject.com/en/1.11/ref/csrf/#acquiring-the-token-if-csrf-use-sessions-is-false
+        // https://docs.djangoproject.com/en/1.11/ref/csrf/#setting-the-token-on-the-ajax-request
+        var csrftoken = Cookies.get('csrftoken');
+        function csrfSafeMethod(method) {
+            return (/^(GET|HEAD|OPTIONS|TRACE)$/.test(method));
+        }
+        $.ajaxSetup({
+            beforeSend: function(xhr, settings) {
+                if (!csrfSafeMethod(settings.type) && !this.crossDomain) {
+                    xhr.setRequestHeader("X-CSRFToken", csrftoken);
+                }
+            }
+        });
+
+        // Create BotUI.
+        var botui = new BotUI('my-botui-app');
+
+        botui.message.add({
+          content: 'Hello! Ready to get started? Type "start."'
+        });
+
+        // Listen for keypress event (i.e., hitting "enter")
+        $("textarea").keypress(function(event) {
+            // Did the user press the "enter" key?
+            if (event.which === 13) {
+                // Ignore the default function of the "enter" key
+                event.preventDefault();
+                
+                $user_input = this.value
+                
+                if ($user_input) {
```

Comment:
this seems like a good place to break this out into a function call. generally its a good idea to keep event calls short and simple